tests/runme/RUNNING_CI.md

```diff
@@ -0,0 +1,188 @@
+```
+
+Check the created identity pools by running the following command:
+
+```sh
+gcloud iam workload-identity-pools list --location="global"
+```
+
+Now you have your cluster configured properly, configure the following GitHub Action Environment Variables (they are not secrets):
+
+``` { mimeType="text/plain" }
+- GCLOUD_WORKLOAD_IDENTITY_PROVIDER: "projects/<project-id>/locations/global/workloadIdentityPools/<pool-name>/providers/<provider-name>"
+- GCLOUD_SERVICE_ACCOUNT: "<name>@<project-name>.iam.gserviceaccount.com"
+- CLUSTER_LOCATION: "<cluster-location>" e.g us-central1-c
+- CLUSTER_NAME: "<YOUR CLUSTER NAME>"
+```
+
+## Authenticate in Google Cloud GitHub Action
+
+Use [google-github-actions/auth@v1](https://github.com/google-github-actions/auth) for authenticating in Google Cloud from your GitHub Action.
+
+ ``` { mimeType=text/x-yaml}
+ - id: 'gcloud-auth'
+  name: 'Authenticate to Google Cloud'
+  uses: 'google-github-actions/auth@v1'
+  with:
+    workload_identity_provider: ${{ vars.GCLOUD_WORKLOAD_IDENTITY_PROVIDER }}
+    service_account: ${{ vars.GCLOUD_SERVICE_ACCOUNT }}
+```
+
+### Allow the CI to interact with the Kubernetes Cluster
+
+Use [google-github-actions/get-gke-credentials@v1](https://github.com/google-github-actions/get-gke-credentials)
+
+ ``` { mimeType=text/x-yaml}
+- id: 'get-credentials'
+  uses: 'google-github-actions/get-gke-credentials@v1'
+  with:
+     cluster_name: ${{ vars.CLUSTER_NAME }}
+     location: ${{ vars.CLUSTER_LOCATION }}
+```
+
+### Setup kubectl
+
+Linkerd requires [kubectl](https://kubernetes.io/docs/tasks/tools/), the Kubernetes command line tool. Microsoft provides a GitHub Action for installing it via [azure/setup-kubectl@v3](https://github.com/Azure/setup-kubectl)
+
+ ``` { mimeType=text/x-yaml}
+- uses: azure/setup-kubectl@v3
+  with:
+  version: 'latest'
+```
+
+If you want to ensure Kubectl is correctly installed in your GitHub Action, you can add the following step:
+
+``` { mimeType=text/x-yaml}
+- name: Check kubectl version
+  run: |
+    kubectl version --short
+```
+
+
+### Install Linkerd
+You can install Linkerd via CURL, and add it to the PATH, so the binary is available for other steps.
+
+
+``` { mimeType=text/x-yaml}
+- name: Install Linkerd
+  run: |
+     curl --proto '=https' --tlsv1.2 -sSfL https://run.linkerd.io/install | sh
+- name: Add Linkerd to PATH
+  run: |
+    echo "$PATH:/home/runner/.linkerd2/bin" >> $GITHUB_PATH
+
+- name: Validate Linkerd install
+run: linkerd version
+```
+
+
+### Install Runme
+
+The Runme CLI can be installed via **go install**
+
+``` { mimeType=text/x-yaml}
+- uses: actions/setup-go@v3
+  with:
+   go-version: '>=1.17.0'
+   - run: go version          
+   - name: Install runme
+     run: |
+        go install github.com/stateful/runme@latest
+```
+
+You can check the Runme version and is installed adequately via the following action:
```

Comment:
Should we include something about running with the grpc runner? See my branch for the relevant changes

Reply:
Added